### PR TITLE
Drop EL6 and Debian 8 and Ubuntu 16.04 support

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
@ekohl Can you review this? Since we are releasing a new major version this should go in.

File: metadata.json [hunk 1]
```diff
@@ -35,36 +35,26 @@
...
       ]
     },
     {
       "operatingsystem": "Scientific",
       "operatingsystemrelease": [
-        "6",
         "7"
       ]
     },
-    {
```
Comment (ekohl) [inline metadata.json:58]: Can we keep Fedora in and add 33/34?

**Review by ekohl — CHANGES_REQUESTED** [review]
Can we also drop Ubuntu 16.04 (EOL since April 2021) while we do a major bump?

File: metadata.json [hunk 2]
```diff
@@ -35,44 +35,39 @@
...
         "7"
       ]
     },
     {
       "operatingsystem": "Fedora",
       "operatingsystemrelease": [
-        "32"
-      ]
+        "33",
+        "34"
```
Comment (ekohl) [inline metadata.json:59]:
```suggestion
        "34"
      ]
```

**Comment by ehelms** [comment]
Removed the changes to Fedora given the testing on 33 and 34 does not work yet.

**Review by ekohl — APPROVED** [review]

**Pull request merged**